README.md

```diff
@@ -575,7 +577,7 @@ Translations for further locales can be added quite easily:
   "Add a new locale". Check that no translations have been lost (except ones which are no
```

Comment:
For consistency it would make sense to cover this occurrence as well.

Reply:
I missed this one. Good catch.